downstream/diarization/config.yaml

```diff
@@ -26,11 +26,15 @@ downstream_expert:
 
   loaderrc:
     num_workers: 8
-    train_batchsize: 32
+    train_batchsize: 8
     eval_batchsize: 1
     train_dir: ./downstream/diarization/data/train
     dev_dir: ./downstream/diarization/data/dev
     test_dir: ./downstream/diarization/data/test
+  
+  scorerc:
+    score_dir: ./downstream/diarization/wav2vec2_scoring
```

Comment:
Is it possible we don't use this extra path and directly dump the h5 file into `expdir` which is an argument of DownstreamExpert (specified in command line with --name or --expdir)?
Since user will like to do the following thing in a shell script:
```
python3 -m train --expdir [exp_path]
python3 -m evaluate -t test -e [exp_path]/best-dev.ckpt  # This will share the expdir with training stage
./score.sh [exp_path]
```

Reply:
Also, it will be nice if `score.sh` takes arguments in the above format, which can enable a faster pipeline to evaluate lots of upstream representation.

Reply:
Definitely make sense. Thanks for the suggestion.